ros2/naive/src/node.cpp

```diff
@@ -0,0 +1,13 @@
+#include "naive/incrementer.hpp"
+#include <rclcpp/rclcpp.hpp>
+
+int main(int argc, char **argv) {
+
+	rclcpp::init(argc,argv);
+	auto const node = std::make_shared<rclcpp::Node>("incrementer");
+	Incrementer incrementer{node, "/in", "/out"};
+	rclcpp::spin(node);
+	rclcpp::shutdown();
+
+	return 0;
```

Comment:
For what it's worth, this is unnecessary. It's specified that if a program reaches the end, it will always return 0 so it's safe to remove this.